buildSrc/src/main/kotlin/JavaBasedProjectConventions.kt

```diff
@@ -30,28 +30,24 @@ fun Project.javaBasedProjectConventions() {
     }
 
     dependencies {
-        "api"(kotlin("stdlib", "1.5.32"))
+        "api"(kotlin("stdlib"))
 
-        "testImplementation"(libraries.junit)
-        "testImplementation"(libraries.truth)
+        "testImplementation"(findLibrary("junit-jupiter"))
+        "testImplementation"(findLibrary("truth"))
     }
 
     tasks.withType<KotlinCompile> {
         kotlinOptions {
             allWarningsAsErrors = true
-            jvmTarget = "1.8"
-            freeCompilerArgs = listOf("-Xinline-classes")
-            languageVersion = "1.5"
-            apiVersion = "1.5"
+            freeCompilerArgs = listOf("-Xjvm-default=all")
```

Comment:
what does jvm-default do again?

Reply:
Tells the Kotlin compiler to generate default methods for non-abstract Kotlin interface members. https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.jvm/-jvm-default/

This was needed for ABI compatibility for drop-in compatibility before I changed the package naming scheme of the runtime and can probably be removed.

Reply:
https://github.com/open-toast/protokt/pull/168/commits/c90f80423981a28ec3cbae8ea55c57a689c35483